src/comps/theme.jsx

```diff
@@ -0,0 +1,22 @@
+import { Brightness7, Brightness4 } from "@mui/icons-material";
+import { IconButton, useTheme } from "@mui/material";
+import { Box } from "@mui/system";
+import { useContext } from "react";
+import { createContext } from "react";
+
+const ColorModeContext = createContext({ toggleColorMode: () => {} });
+
+const Theme = () => {
+  const theme = useTheme();
+  const colorMode = useContext(ColorModeContext);
+  return (
+    <Box sx={{ position: "fixed", top: 10, left: 10, backgroundColor: theme.palette.background.default, borderRadius: "100%" }}>
```

Comment:
The Icon now has a Background to see it better
<img width="44" alt="image" src="https://user-images.githubusercontent.com/68782035/206142795-22e0d2b0-dbb7-4a85-8494-b5634e2acf26.png">